### ISSUE TITLE
KPI Setting Workshop (C+L team)

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#18380]
This is a request to schedule a workshop to help establish KPIs for the Content and Localization Team.

Attached is the Pre-workshop Planning doc:  https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/content-localization

**Comment by joanneesteban** [comment]
Hi @SKasimow - please let us know if you need anything else, or if you have any feedback on how we could have improved our services for you and your team during this session!

Moving this ticket to Review on the Analytics side.

**Comment by SKasimow** [comment]
@joanneesteban Thanks Joanne! I took an action to add in the engagement metrics we discussed. Should I add them to the reporting dashboard ticket Ben is working on? Or would I add them here to have a separate reporting dashboard for KPIs?

Also, once we are aligned around KPIs, is it up to us to set goals (ie: achieve 5% increase in page views)? Or is that something your team would help establish?

Thanks again!
